Project1/main.cpp

```diff
@@ -0,0 +1,21 @@
+
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+#define STR_SIZE 3
+
+int main() {
+    string str;
+    for (int i = 1; i < 12; i++) {
+        for (int j = 1; j < 12; j++) {
+            str = to_string(i * j);
```

Comment:
std::cout is non-discriminatory and is willing to receive integers and not only strings! :)
No need to construct a string before using it.

Reply:
As for having all numbers with equal spacing, you can check out std::setw and figure out how it can help you accomplish what you used std.resize for